### ISSUE TITLE
Replace the "completed" checkbox with something like PrioritySelector.

### ISSUE THREAD

**Issue by hobarrera** [pimutils/todoman#265]
This'll allow changing status to cancelled/in-progress/etc via the UI (currently, some statuses are only possible via the cli interface).

**Comment by untitaker** [comment]
Generally I would recommend against implementing each peculiarity of iCalendar... boolean "completed" value is the 80% usecase.

**Comment by hobarrera** [comment]
Yeah, that makes sense. TBH, I wanted CLI/UI parity, but if nobody needs it, I'll skip that.

I *will* however, hide the `completed` checkbox when running `todo new`, since it doesn't make sense to create a completed todo.
